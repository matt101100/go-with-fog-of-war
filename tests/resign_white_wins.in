place A1
place A2
resign
