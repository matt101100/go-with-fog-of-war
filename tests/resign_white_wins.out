White wins!
A1A2
Thank you for playing!
